ts/parts/PlotLineOrBand.ts

```diff
@@ -229,18 +229,17 @@ H.PlotLineOrBand.prototype = {
         // Set the presentational attributes
         if (!axis.chart.styledMode) {
             if (isLine) {
-                attribs.stroke = color as any;
+                attribs.stroke = (color as any) || '${palette.neutralColor40}';
                 attribs['stroke-width'] =
-                    (options as Highcharts.AxisPlotLinesOptions).width;
+                    (options as Highcharts.AxisPlotLinesOptions).width ||
```

Comment:
Just for a bullet proof solution, I would go with `pick()` instead of `||`. Setting `width=0` is the easiest way to hide/show plotLine

Reply:
`pick()` is not type safe (at the moment).